sql/core/src/main/scala/org/apache/spark/sql/internal/SessionState.scala

```diff
@@ -38,10 +38,7 @@ import org.apache.spark.sql.util.ExecutionListenerManager
 
 /**
  * A class that holds all session-specific state in a given [[SparkSession]].
- * @param sparkContext The [[SparkContext]].
- * @param sharedState The shared state.
  * @param conf SQL-specific key-value configurations.
- * @param experimentalMethods The experimental methods.
```

Comment:
These comments are adding little or no value. We should remove or make them more detailed, which would you prefer? If the latter, what's a good doc for shared state and experimental methods?